ddtrace/tracer/option.go

```diff
@@ -298,6 +301,12 @@ type config struct {
 
 	// logDirectory is directory for tracer logs specified by user-setting DD_TRACE_LOG_DIRECTORY. default empty/unused
 	logDirectory string
+
+	// tracingAsTransport specifies whether the tracer is running in transport-only mode, where traces are only sent when other products request it.
+	tracingAsTransport bool
+
+	// traceRateLimit specifies the rate limit for traces.
+	traceRateLimit float64
```

Comment:
Same here, would be dope to mention the unit... In ideal situations, the unit would even be part of the field name (e.g, `timeoutSeconds` versus `timeoutMillis` makes it quite obvious on all usage locations what this is at, which reduces the risk of mis-handling).

Reply:
Done in https://github.com/DataDog/dd-trace-go/pull/3033/commits/4bb0581fe708ab53f71e56c4ac89f2e067526ffe